src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01.mdx

```diff
@@ -85,7 +85,8 @@ New Relic has provided the following resources to assist with these updates:
 * [Install the newest helm charts for the Kubernetes Plugin](https://github.com/newrelic/helm-charts/tree/master?tab=readme-ov-file#Installthecharts)
 * [Install the Kubernetes integration](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/)
 * [Fluent Bit plugin for log forwarding](/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding/)
-
+* [Check the Fluent Bit versions being used on your host](src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure/#search)
```

Comment:
```suggestion
* [Check the Fluent Bit versions being used on your host](/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure/#search)
```

Reply:
@paperclypse although I fixed the relative path, the anchor doesn't seem valid. Could you check this?

Reply:
I checked it. It links to a collapser. It worked from me in production: https://docs.newrelic.com/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure/#search